nextflow_schema.json

```diff
@@ -476,46 +479,48 @@
                     "fa_icon": "fas fa-layer-group"
                 },
                 "arg_fargene_savetmpfiles": {
                     "type": "boolean",
                     "fa_icon": "fas fa-save",
-                    "description": "Specify to save intermediate temporary files to --outdir",
-                    "help_text": "fARGene produces a lot of additional temporary files which in most cases won't be useful.\n\nBy specifying this parameter, the directories `tmpdir/`, `hmmsearchresults/` and `spades_assemblies/` will be also saved in the output directory for closer inspection by the user, if necessary."
+                    "description": "Specify to save intermediate temporary files to results directory.",
+                    "help_text": "fARGene generates many additional temporary files which in most cases won't be useful and thus by default are not saved to the pipeline's result directory.\n\nBy specifying this parameter, the directories `tmpdir/`, `hmmsearchresults/` and `spades_assemblies/` will be also saved in the output directory for closer inspection by the user, if necessary."
                 },
                 "arg_fargene_score": {
                     "type": "number",
-                    "help_text": "The threshold score for a sequence to be classified as a (almost) complete gene. If not pre-assigned, it is assigned by the hmm_model used based on the trade-off between sensitivity and specificity. For more details see [code documentation](https://github.com/fannyhb/fargene/blob/master/fargene_analysis/fargene_analysis.py)\n\n> Modifies tool parameter(s):\n> -  fARGene: `--score`",
-                    "description": "The threshold score for a sequence to be classified as a (almost) complete gene",
+                    "help_text": "The threshold score for a sequence to be classified as a (almost) complete gene. If not pre-assigned, it is assigned by the hmm_model used based on the trade-off between sensitivity and specificity.\n\nFor more details see [code documentation.](https://github.com/fannyhb/fargene/blob/master/fargene_analysis/fargene_analysis.py)\n\n> Modifies tool parameter(s):\n\n> - fARGene: `--score`",
+                    "description": "The threshold score for a sequence to be classified as a (almost) complete gene.",
                     "fa_icon": "fab fa-creative-commons-zero"
                 },
                 "arg_fargene_minorflength": {
-                    "type": "string",
-                    "default": "'90'",
```

Comment:
I think I put it as a string because it was erroring out as a number

Reply:
what was the error as anumber?

Reply:
I dont remember :( have to check it again

Reply:
Ok it does work with integer